server/lib/scanners/baseScanner.ts

```diff
@@ -283,6 +285,10 @@ class BaseScanner<T> {
               ? MediaStatus.PARTIALLY_AVAILABLE
               : !season.is4kOverride && season.processing
               ? MediaStatus.PROCESSING
+              : settings.main.removeUnmonitoredFromRequestsEnabled &&
+                !season.monitored &&
+                season.episodes == 0
+              ? MediaStatus.UNKNOWN
```

Comment:
Why is this change necessary? Isn't the Sonarr scanner job enough?

Reply:
It seems to be necessary. It **is** part of Sonarr scanner job though!
- At the beginning of the job I check about the SeasonRequest from the Request (and "clean the request")
- And here I check about the Status of the Season, as every Season is parsed and it's Status checked/updated